src/api/sendUpdateMessage.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express';
+import { Subscriber } from '../entities/subscriber';
+import messageUsers from '../slack/utilities/messageUsers';
+import logger from '../logger';
+
+export default async function sendUpdateMessage(req: Request, res: Response): Promise<void> {
+  const { message, secret } = req.body;
+
+  // TODO: Implement auth
+  if (secret !== 'some secret') {
```

Comment:
For now I would just say use `process.env.ADMIN_SECRET` and we can use that everywhere

Reply:
Eventually, I'll have this in the config, but that'll work for now